elisp-bytecode.texi

```diff
@@ -1224,76 +1323,128 @@ Call @code{current-buffer} with no arguments.
 @kindex set-buffer
 Call @code{set-buffer} with one argument.
 
+@node save-current-buffer-1
```

Comment:
I don't think we should call this "save-current-buffer-1", since "byte-save-current-buffer" is what bytecomp.el uses.

Reply:
Ok change, and will push soon.